### DISCUSSION TITLE
input-group with floating label

### DISCUSSION THREAD

**Discussion by uebdigital** [twbs/bootstrap#37870] [Q&A]
I'm trying to use input group with floating label but I'm having  problems with full width width with bootstrap 5.2...

						<div class='col-md-5 mx-auto'>
							<form id='login'>		
							<h1 class="h3 mb-3 fw-normal">Indique os seus dados</h1>
							
							
							<div class="form-floating">
								<input type="text" class="form-control" id="login_contacto" autocomplete="username" placeholder="name@example.com">
								<label for="login_contacto">Email ou telemóvel</label>
							</div>

							<div class="input-group mb-3">
								<div class="form-floating">
								    <input type="password" class="form-control" id="floatingInputGroup1" placeholder="Username">
								    <label for="floatingInputGroup1">Username</label>
								</div>
							  	<span class="input-group-text"><i class="bi bi-eye-slash-fill"></i></span>  
							</div>

							<div class="checkbox mb-3 mt-3">
								<label>
								<input type="checkbox" value="remember-me"> Lembrar-me neste dispositivo
								</label>
							</div>
							<button class="w-100 btn btn-lg btn-primary" type="submit">Autenticar</button>
							<p class="mt-5 mb-3 text-muted">&copy; 2020–<?php date_default_timezone_set('Europe/Lisbon'); echo date("Y"); ?> E-Usados.pt - Todos os direitos reservados</p>
							</form>	
						</div>

Ideas? Someone?

**Comment by uebdigital** [comment]
No one can help on this???

**Comment by timface** [comment]
Hi @uebdigital

I'm not entirely sure what you are wanting to achieve but if you want your input field to stretch to full width while inside an input-group then try this css:
`.input-group > .form-floating {
    flex-grow: 1;
}`

**Comment by tbolt** [comment]
Like @timface suggested you can add a utility class to that `form-floating` so your input group would become-
```
<div class="input-group mb-3">
  <div class="form-floating flex-grow-1">
    <input
      type="password"
      class="form-control"
      id="floatingInputGroup1"
      placeholder="Username"
    />
    <label for="floatingInputGroup1">Username</label>
  </div>
  <span class="input-group-text"><i class="bi bi-eye-slash-fill"></i></span>
</div>
